packages/usdk/packages/upstreet-agent/packages/react-agents/devices/video-input.mjs

```diff
@@ -1,7 +1,7 @@
 import { EventEmitter } from 'events';
 import child_process from 'child_process';
-import Jimp from 'jimp';
-const { intToRGBA } = Jimp;
+import { Jimp } from 'jimp';
+import { intToRGBA } from '@jimp/utils';
```

Comment:
I think this needs to be added to the package deps. In `pnpm` it will not be hoisted or accessible otherwise.